src/common/app_metrics.cpp

```diff
@@ -23,7 +23,11 @@ void Buddy::Metrics::RecordRuntimeStats() {
     if (ticks_diff(ticks_ms(), last_recorded_ticks) > 3000) {
         int count = uxTaskGetSystemState(task_statuses, sizeof(task_statuses) / sizeof(task_statuses[1]), NULL);
         for (int idx = 0; idx < count; idx++) {
-            size_t s = malloc_usable_size(task_statuses[idx].pxStackBase);
+            char *stack_base = (char *)task_statuses[idx].pxStackBase;
```

Comment:
`const char *`

Reply:
OK, I'll do it, but I still need to cast the `const` out later in the call to `malloc_usable_size(...)` since it accepts `void *`. That's why it's not const now.

Reply:
Done.